C/include/c4BlobStore.h

```diff
@@ -138,7 +142,7 @@ CBL_CORE_API int64_t c4stream_getLength(C4ReadStream*, C4Error* C4NULLABLE) C4AP
 CBL_CORE_API bool c4stream_seek(C4ReadStream*, uint64_t position, C4Error* C4NULLABLE) C4API;
 
 /** Closes a read-stream. (A NULL parameter is allowed.) */
-CBL_CORE_API void c4stream_close(C4ReadStream* C4NULLABLE) C4API;
+CBL_CORE_API void c4stream_close(C4ReadStream* C4NULLABLE) C4API;  // NOLINT(readability-redundant-declaration)
```

Comment:
It's a redundant declaration because `c4stream_close` is declared in both `c4Base.h` and `c4BlobStore.h`. One option is to remove it from `c4BlobStore.h`.

Reply:
That should be done if it hasn't already